core/test/acc_text/nlg/core_test.clj

```diff
@@ -17,16 +17,16 @@
 
 (deftest ^:integration list-generation
   (let [context {:data       {"product" "product." "fridge" "fridge."}
-                 :dictionary {"fridge"  #:acc-text.nlg.dictionary.item{:id       "03393900-ddbf-426f-b267-f100d09824c0"
-                                                                       :key      "fridge"
-                                                                       :category "N"
-                                                                       :language "Eng"
-                                                                       :forms    ["fridge" "fridges"]}
-                              "product" #:acc-text.nlg.dictionary.item{:id       "9399a059-f5a5-4443-bee7-a096d284a85d"
-                                                                       :key      "product"
-                                                                       :category "N"
-                                                                       :language "Eng"
-                                                                       :forms    ["product" "products"]}}}]
+                 :dictionary {["fridge" "N"]  #:acc-text.nlg.dictionary.item{:id       "03393900-ddbf-426f-b267-f100d09824c0"
```

Comment:
having to define category two times (as a key and as a :category) is a bit redundant?

Reply:
Key ("fridge") is also defined two times. They both matter when looking up dictionary items now since there may be words with similar keys but different categories.

Reply:
and "fridge" will look better in UI instead of "fridge_N"